id: PYSEC-2019-110
package:
  name: pillow
  ecosystem: PyPI
summary: Vulnerability in pillow
details: An issue was discovered in Pillow before 6.2.0. When reading specially crafted
  invalid image files, the library can either allocate very large amounts of memory
  or take an extremely long period of time to process the image.
affects:
  ranges:
  - type: ECOSYSTEM
    fixed: 6.2.0
references:
- type: WEB
  url: https://pillow.readthedocs.io/en/latest/releasenotes/6.2.0.html
- type: WEB
  url: https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/LYDXD7EE4YAEVSTNIFZKNVPRVJX5ZOG3/
- type: WEB
  url: https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/EMJBUZQGQ2Q7HXYCQVRLU7OXNC7CAWWU/
- type: WEB
  url: https://usn.ubuntu.com/4272-1/
- type: WEB
  url: https://access.redhat.com/errata/RHSA-2020:0566
- type: WEB
  url: https://www.debian.org/security/2020/dsa-4631
- type: WEB
  url: https://access.redhat.com/errata/RHSA-2020:0580
- type: WEB
  url: https://access.redhat.com/errata/RHSA-2020:0578
- type: WEB
  url: https://access.redhat.com/errata/RHSA-2020:0683
- type: WEB
  url: https://access.redhat.com/errata/RHSA-2020:0681
- type: WEB
  url: https://access.redhat.com/errata/RHSA-2020:0694
aliases:
- CVE-2019-16865
modified: "2020-02-18T16:15:00Z"
published: "2019-10-04T22:15:00Z"
